code/include/playfab/InternalMemory.h

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+#include <set>
+#include <queue>
+#include <unordered_set>
+
+namespace PlayFab
+{
+
+//------------------------------------------------------------------------------
+// Allocator
+//------------------------------------------------------------------------------
+template<class T>
+struct Allocator

```

Comment:
is this intended to be a default allocator?

Reply:
For now it is, yes.  Plan will be to add public methods to configure allocations but that hasn't been done yet.

---
In reply to: [595556635](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r595556635) [](ancestors = 595556635)